packages/airnode-node/src/api/index.ts

```diff
@@ -244,12 +247,19 @@ async function processSuccessfulApiCall(
 
   const response = goExtractAndEncodeResponse.data;
 
+  const airnodeWalletPrivateKey = getEnvValue('AIRNODE_WALLET_PRIVATE_KEY');
+  if (!airnodeWalletPrivateKey) {
+    const errorMessage = 'Missing Airnode wallet private key in environment variables.';
+    const log = logger.pend('ERROR', errorMessage);
+    return [[log], { success: false, errorMessage }];
+  }
```

Comment:
Two things come to mind looking at this:
* This means that the env variable is required (which is fine). In that case, make the Terraform variable required so you can't somehow deploy without it.
* There is no need for this env variable for HTTP gateway (non-signed) requests. So, I would make this check only where is needed and also add the variable (in Terraform) only to handlers that need it.